maintainers/maintainer-list.nix

```diff
@@ -11308,4 +11308,14 @@
     github = "zupo";
     githubId = 311580;
   };
+  zane = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
Fixed, but the list isn't entirely sorted. It's above `zseri`, but the next one is `pulsation`. I hope that's alright.

Reply:
There are some entries at the end that need sorting.